ibis/backends/tests/test_temporal.py

```diff
@@ -578,6 +578,41 @@ def test_to_timestamp(backend, con, unit):
     assert result == expected
 
 
+@pytest.mark.parametrize(
+    'fn',
+    [
+        param(lambda t: t.date_string_col.to_timestamp('%m/%d/%y')),
+        param(
+            lambda t: t.date_string_col.to_timestamp(
+                '%m/%d/%y', timezone='UTC'
```

Comment:
Try "Pacific/Marquesas"? :)

Reply:
Yes! To test this properly, we will need non-UTC timezones.

Reply:
"Asia/Tehran" around DST jumps can be fun.